### PR TITLE
Feat new python versions

### PR DESCRIPTION
**Pull request by wirthual**
- Fixed typo in readme
 - Set protobuf to lower version to avoid installation error (#65 )
 - Updated lock
 - Use latest available patch per python version
 - Update github action name for docs

### CODE REVIEW

File: .github/workflows/runtests.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.6.7","3.7", "3.8", "3.9"]
+        python-version: ["3.9.13","3.9.10","3.9.6","3.9.5","3.9.4","3.9.3","3.9.1","3.9.0","3.8.13","3.8.12","3.8.10","3.8.9","3.8.8","3.8.7","3.8.6","3.8.5","3.8.3","3.8.2","3.8.1","3.8.0","3.7.13","3.7.12","3.7.11","3.7.9","3.7.8","3.7.6","3.7.5","3.7.1","3.6.12","3.6.11","3.6.10","3.6.9"]
```
Comment (jugmac00) [inline .github/workflows/runtests.yml:10]: It is not necessary to test Python versions on a patch level.

This gives basically nothing, but only increases the test time and also the resource consumption.

Comment (lukaspanni) [reply .github/workflows/runtests.yml:10]: I agree, testing patches is not really necessary

**Review by lukaspanni — CHANGES_REQUESTED** [review]
I don't really like testing so much different python versions.
That not only increases test and thus feedback time and also increases the risk of failure due to any API that is currently not available (as we use real API-calls in our tests).

File: .github/workflows/runtests.yml (hunk 1 above)
Comment (wirthual) [reply .github/workflows/runtests.yml:10]: Makes sense. Will keep the latest patch for each minor version 👍

**Pull request merged**